src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -20,7 +20,12 @@ const StyledCheckBoxIcon = styled.svg`
   box-sizing: border-box;
   stroke-width: ${props => props.theme.checkBox.check.thickness};
   stroke: ${props =>
-    normalizeColor(props.theme.checkBox.color || 'control', props.theme)};
+    props.toggle
+      ? normalizeColor(
+          props.theme.checkBox.toggle.icon.color || 'transparent',
```

Comment:
I think there might be a better way to assign the icon and the icon color.
Currently, when the icon isn't used or colored, it is spamming the dom with an unneeded SVG, and a redundant nested `span` element.
I think the icon shouldn't be applied on the DOM if the user wasn't asking for it.